workflow/cylc/sampled-examples/util/template.py

```diff
@@ -0,0 +1,24 @@
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#   "pandas",
+#   "typer",
+# ]
+# ///
+
+import pathlib
+
+import pandas
+import typer
+
+
+def main(datafile: pathlib.Path, index_col: str, index_val: str):
+    df = pandas.read_csv(datafile, index_col=index_col)
+    r = df.loc[index_val]
+    print(
+        f"""--icp {r.startdate} --fcp {r.enddate} --set BBOX="{r.left_x},{r.lower_y},{r.right_x},{r.top_y}" --set "LOCATION='{r.location}'" """
+    )
```

Comment:
```suggestion
    template_with_good_name = (
    "--icp {startdate} "
    "--fcp {enddate} "
    '--set BBOX="{left_x},{lower_y},{right_x},{top_y}" '
    '--set "LOCATION=\'{location}\'"'
)

    filledin_template = template_with_good_name.format(**r.__dict__)

    print(filledin_template)

```

Reply:
Thanks for this feedback!
I think that splitting the string across several lines is probably a good move. 
I think making the thing which does the template conversion into its own testable function might be a good compromise. I've updated this in a new version.